src/model_finetuning/metric.py

```diff
@@ -0,0 +1,22 @@
+import torch
+import torch.nn as nn
+from torch import Tensor
+
+
+class MCRMSELoss(nn.Module):
+
+    def __init__(self):
+        super(MCRMSELoss, self).__init__()
+        self.mse = nn.MSELoss(reduction='none')
+
+    def forward(self, y_pred: Tensor, y_true: Tensor):
+        mse = self.mse(y_pred, y_true).mean(0)  # column-wise mean
```

Comment:
just for reliability, I would suggest you to add asserts in the beginning of both functions, where you check tensor shapes.

otherwise, if y_pred has a shape of [64, ], and y_true has a shape of [64, 1], you are at risk to get a shape of self.mse(y_pred, y_true) of [64, 64]

Reply:
I do not this that it is good practice to insert assert in PyTorch code, right now I simply added docstring, describing shapes on input tensors